docs/src/main/asciidoc/core.adoc

```diff
@@ -61,9 +76,45 @@ If it does not serve your project needs, this behavior can be changed by setting
 
 |spring.cloud.aws.credentials.profile.path
 |`~/.aws/credentials`
-|The file path where the profile configuration file is located. Defaults to `~/.aws/credentials` if value is not provided
+|The file path where the profile configuration file is located. Defaults to `~/.aws/credentials` if a value is not provided
 |===
 
+==== StsWebIdentityTokenFileCredentialsProvider
+
+The `StsWebIdentityTokenFileCredentialsProvider` allows your application to assume an AWS IAM Role using a web identity token file, which is especially useful in Kubernetes and AWS EKS environments.
+
+===== Prerequisites
+1. Create a role that you want to assume.
+2. Create a web identity token file for your application.
+
+In EKS, please follow this guide to set up service accounts https://docs.aws.amazon.com/eks/latest/userguide/pod-configuration.html
+
+The `StsWebIdentityTokenFileCredentialsProvider` support is optional, so you need to include the following Maven dependency:
+[source,xml,indent=0]
+----
+<dependency>
+	<groupId>software.amazon.awssdk</groupId>
+	<artifactId>sts</artifactId>
+</dependency>
+----
+
+===== Configuring
+In EKS no additional configuration is required as the service account already configures the correct environment variables; however, they can be overridden.
```

Comment:
We should have here a table with available properties and i don't think we need to document system properties and env variables used by AWS SDK. Maybe we can link to their official docs.

Reply:
Agree

Reply:
I think saying `(falls back to SDK default)`. What do you think @maciejwalkowiak